pom.xml

```diff
@@ -45,6 +45,12 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.springframework</groupId>
```

Comment:
Zróbmy to na Feignie - na pewno Ci się przyda, a webfluxy by Ci zrobiły ostry midnfak :D